django_tablib/admin/__init__.py

```diff
@@ -1,111 +1,132 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
 import datetime
 import django
-
+from distutils.version import LooseVersion
 from django.contrib import admin
-from django.contrib.admin.views.main import ChangeList
 from django.core.urlresolvers import reverse
 from django.http import Http404
 
+
 from django_tablib.base import mimetype_map
 from django_tablib.views import export
 
+from . import actions as django_tablib_actions
+
 try:
     # Removed in Django 1.6; default to stdlib.
     from functools import update_wrapper
 except ImportError:
     from django.utils.functional import update_wrapper
 
+INVALID_EXPORT_FORMAT_ERROR_MSG = (
+    "{0} is not a valid export format, "
+    "please choose from the following options: {1}")
+
 
 class TablibAdmin(admin.ModelAdmin):
     change_list_template = 'tablib/change_list.html'
     formats = []
     headers = None
     export_filename = 'export'
+    # enable Export to _format_ admin actions by default, this allows to export
+    # only selected items.
+    enable_admin_actions = True
 
     def __init__(self, *args, **kwargs):
-        for format in self.formats:
-            if format not in mimetype_map:
-                msg = "%s is not a valid export format, please choose" \
-                    " from the following options: %s" % (
-                        format,
-                        ', '.join(mimetype_map.keys()),
-                    )
-                raise ValueError(msg)
+        for export_format in self.formats:
+            if export_format not in mimetype_map:
+                raise ValueError(INVALID_EXPORT_FORMAT_ERROR_MSG.format(
+                    export_format, ', '.join(mimetype_map.keys())))
         super(TablibAdmin, self).__init__(*args, **kwargs)
 
+    def get_info(self):
+        """
+        Get model info in a form of (app_label, model_name).
+        """
```

Comment:
``module_name`` is removed in ``Django`` 1.8, 1.7 produces deprecation warnings, but 1.6 doesn't have ``model_name`` (afaik), and to not to make same code twice i did a small helper function. Not sure that it should have such detailed info in its description...